internal/workload/podman/podman.go

```diff
@@ -282,14 +283,15 @@ func (p *podman) UpdateSecret(name, data string) error {
 	return p.CreateSecret(name, data)
 }
 
-func (p *podman) GenerateSystemdService(podName string, monitoringInterval uint) (service.Service, error) {
-	useName := true
-	report, err := generate.Systemd(p.podmanConnection, podName, &generate.SystemdOptions{UseName: &useName, RestartSec: &monitoringInterval})
+func (p *podman) GenerateSystemdService(podId string, monitoringInterval uint) (service.Service, error) {
+	//report, err := generate.Systemd(p.podmanConnection, podId, &generate.SystemdOptions{RestartSec: &monitoringInterval})
```

Comment:
Please remove commented-out code.

Reply:
Ah, thanks. I've found bug in podman, that if you pass empty podId, it crashes, so that's why I added those here and I forgot to remove.